CMakeLists.txt

```diff
@@ -289,6 +289,13 @@ if (USE_ASAN)
     set (CMAKE_LINKER_FLAGS_DEBUG "${CMAKE_STATIC_LINKER_FLAGS_DEBUG} -fsanitize=address")
 endif()
 
+if (MSVC)
```

Comment:
nit: this should be in cmake/MiscCheck.cmake? We have a much better commented flags setting here: https://github.com/pytorch/pytorch/blob/master/cmake/MiscCheck.cmake#L157

Reply:
Good point - moving there.